assets/bindata_generator.go

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright 2023 VMware, Inc.
```

Comment:
the copyright stable format is start with "Copyright 2022-2023 VMware, Inc."

Reply:
For a new created file, this is the correct format. The format you provided is for the files which were created before current year.